docker-compose.yaml

```diff
@@ -72,7 +72,7 @@ services:
       - LDAP_PASS=${LDAP_PASS}
     depends_on:
       - ledger-sync
-      - rethink
+      # - rethink
```

Comment:
If we really don't need these, then the lines can be deleted.

Reply:
my mistake, I'll rebase